solaris/__init__.py

```diff
@@ -1,4 +1,4 @@
-from . import data, utils, vector  # noqa: F401
+from . import data, tile, utils, vector  # noqa: F401
```

Comment:
I had to add # noqa: F401 back in or else the linter failed (I think your original change may have added `tile,` and removed the comment